libs/payments/ui/src/lib/client/components/LoadingSpinner/index.tsx

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use client';
+
+import { Localized } from '@fluent/react';
+import spinnerImage from '@fxa/shared/assets/images/spinner.svg';
+import Image from 'next/image';
+
+/**
+ * Next.js implementation of a loading spinner for use within client components
+ */
+export function LoadingSpinner() {
+  return (
+    <div
+      id="loading-spinner"
```

Comment:
I don't think this div needs an id?

Reply:
The other instances of this class have it. If we don't need it then I'm happy to remove it!